docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1995 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #66.        print('Window size:', end=' ')
</pre><p>Remove all print statements inside of functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
Sure. I actually planned to remove it after your review. Will do it in next set of commis.